vcf-dumper/vcf-dumper-lib/src/main/java/embl/ebi/variation/eva/vcfdump/BiodataVariantToVariantContextConverter.java

```diff
@@ -38,28 +35,25 @@
     public static final String GENOTYPE_KEY = "GT";
     private final VariantContextBuilder variantContextBuilder;
     private List<VariantSource> sources;
-    private CellbaseWSClient cellbaseClient;
-    private String regionSequence;
+    private Set<String> studies;
     private Map<String, Map<String,String>> filesSampleNamesEquivalences;
     private static final int NO_CALL_ALLELE_INDEX = 2;
 
     protected static final Pattern genotypePattern = Pattern.compile("/|\\|");
 
-    public BiodataVariantToVariantContextConverter(CellbaseWSClient cellbaseWSClient) {
-        this(null, cellbaseWSClient, null);
+    public BiodataVariantToVariantContextConverter() {
+        this(null, null);
     }
 
-    public BiodataVariantToVariantContextConverter(List<VariantSource> sources, CellbaseWSClient cellbaseWSClient,
-                                                   Map<String, Map<String,String>> filesSampleNamesEquivalences)
+    public BiodataVariantToVariantContextConverter(List<VariantSource> sources, Map<String, Map<String,String>> filesSampleNamesEquivalences)
```

Comment:
Removed default constructor and done some small refactor in VariantExporter to use the constructor with params, instead of the default + calls to set methods. f934bff